Integrations/Flashpoint/Flashpoint.py

```diff
@@ -0,0 +1,1246 @@
+
+    query = r'+type:("url") +value.\*.keyword:"' + url + '"'
+    resp = client.http_request("GET", url_suffix=get_url_suffix(query))
+
+    if isinstance(resp, list):
+        indicators = resp
+    else:
+        indicators = []
+
+    if len(indicators) > 0:
+
+        hr = '### Flashpoint URL reputation for ' + url + '\n'
+        hr += 'Reputation: Malicious\n\n'
+
+        events_details = get_events_from_ioc_resp(indicators)
+
+        hr += tableToMarkdown('Events in which this IOC observed', events_details['events'],
+                              ['Date Observed (UTC)', 'Name', 'Tags'])
+
+        fp_link = client.url + '/home/search/iocs?group=indicator&ioc_type=url&ioc_value=' + encoded_url
+        hr += '\nAll events and details (fp-tools): [{}]({})\n'.format(fp_link, fp_link)
+
+        ec = {outputPaths['url']: {
+            'Name': url,
+            'Flashpoint': {
+                'href': events_details['href']
+            }
+        }, 'DBotScore': {
+            'Indicator': url,
+            'Type': 'url',
+            'Vendor': 'Flashpoint',
+            'Score': 3
+        }}
+
+        ec[outputPaths['url']]['Malicious'] = {
+            'Vendor': 'Flashpoint',
+            'Description': 'Found in malicious indicators dataset'
+        }
+
+        return hr, ec, resp
+
+    else:
+        hr = '### Flashpoint URL reputation for ' + url + '\n'
+        hr += 'Reputation: Unknown\n\n'
+        ec = {
+            'DBotScore': {
+                'Indicator': url,
+                'Type': 'url',
+                'Vendor': 'Flashpoint',
+                'Score': 0
+            }
+        }
+
+        return hr, ec, resp
+
+
+def file_lookup(client, file):
+    """
+    'file' command to lookup a particular file hash (md5, sha1, sha256, sha512)
+
+    :param client:
+    :param file: file
+    :return: command output
+    """
+
+    query = r'+type:("md5", "sha1", "sha256", "sha512") +value.\*.keyword:"' + file + '"'
+    resp = client.http_request("GET", url_suffix=get_url_suffix(query))
+
+    if isinstance(resp, list):
+        indicators = resp
+    else:
+        indicators = []
+
+    if len(indicators) > 0:
+        indicator_type = indicators[0].get('Attribute', {}).get('type')
+        hr = '### Flashpoint File reputation for ' + file + '\n'
+        hr += 'Reputation: Malicious\n\n'
+
+        events_details = get_events_from_ioc_resp(indicators)
+
+        hr += tableToMarkdown('Events in which this IOC observed', events_details['events'],
+                              ['Date Observed (UTC)', 'Name', 'Tags'])
+
+        fp_link = client.url + '/home/search/iocs?group=indicator&ioc_type=md5%2Csha1%2Csha256%2Csha512' \
+                               '&ioc_value=' + urllib.parse.quote(file.encode('utf8'))
+
+        hr += '\nAll events and details (fp-tools): [{}]({})\n'.format(fp_link, fp_link)
+
+        ec = {outputPaths['file']: {
+            indicator_type: file,
```

Comment:
the ec dict and the context path must match.
So in context you defined 'File.SHA256' but in code you export 
```json
{
     "File": {
          "sha256": ....
     }
}
```

SHA256 != sha256 they are not equal.